crates/holochain_conductor_api/src/config/conductor/dpki_config.rs

```diff
@@ -77,11 +79,17 @@ impl DpkiConfig {
 
 impl Default for DpkiConfig {
     fn default() -> Self {
-        Self {
-            dna_path: None,
-            network_seed: DPKI_NETWORK_SEED_TESTING.to_string(),
-            allow_throwaway_random_dpki_agent_key: false,
-            no_dpki: false,
+        cfg_if! {
+            if #[cfg(feature = "dpki")] {
```

Comment:
I see this feature already existed but we talked yesterday about a naming pattern for these. Should this be something like `unstable-dpki`? To make it really clear why it exists and that it will be removed once the feature is stabilised?